style/index.css

```diff
@@ -0,0 +1,113 @@
+.categories-inner-box {
+  gap: 7rem;
+}
+
+.img-header-container {
+  margin: 3rem;
+  gap: 2rem;
+}
+
+.hero-img {
+  width: 60rem;
+}
+
+.content {
+  font-weight: bolder;
+}
+
+.inner-header-content {
+  gap: 1.8rem;
+  margin-top: 2.5rem;
+}
+
+.category-header {
+  margin-top: 7rem;
+  text-align: center;
+}
+
+.categories-inner-box {
+  margin: 5rem;
+}
+
+/* card */
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white-color);
+  width: 40rem;
+  border-radius: 7px;
+  box-shadow: var(--box-shadow);
+}
+
+.basic-card {
+  max-width: 40rem;
+  background-color: var(--white-color);
+}
+
+.basic-card-img-container img {
+  height: 27rem;
+  max-width: 40rem;
```

Comment:
instead of fixed values you could use variables, I've seen you have used width :40rem in many places so using a variable or utility class would be better